docs/walkthrough.rst

```diff
@@ -0,0 +1,201 @@
+error in this case.
+
+All from selected
+^^^^^^^^^^^^^^^^^
+The fourth option is to apply the scripts to all of the selected objects and the children  \
+objects. Selecting a Project and using the option **<all (from selected)>** with result in \
+applying the scripts to the selected project, to the linked datasets and the linked images \
+bellow. Note that this option does not exist for the import of annotation from csv.
+
+Exporting annotations
+---------------------
+
+Exporting annotations creates a .csv file with headers compatible with the import script. \
+It can be used for:
+
+* Export key-value pairs (along with data like namespaces and tags) for modification or \
+  transferring annotations between groups.
+* Export a list of objects with their names and IDs (and excluding key-value pairs by \
+  specifying an unused namespace).
+
+.. figure:: images/annotation_04_export_output.png
+
+   *Exported .csv with only the object ID and name*
+..
+
+Importing annotations
+---------------------
+
+Starting from the file we exported as explained in the previous section, \
+we proceed to edit it within a spreadsheet editor, adding more columns to the csv, \
+You can also insert a row on the top of .csv to specify the namespace for each key:
+
+.. figure:: images/annotation_05_KV_to_import.png
+
+   *Result of populating the csv shown in a text editor*
+..
+
+We added several columns to annotate our dataset with Key-Value pairs \
+following the `REMBI <https://doi.org/10.1038/s41592-021-01166-8>`_ guidelines \
+(after saving the document, our ``TAB`` separator became ``,``).
+
+We proceed and start the script "*Import from CSV*".
+
+.. figure:: images/annotation_06_script_import.png
+
+   *Selection of all the datasets of a project and the Import from CSV script.*
+..
+
+There are many parameters here that could help you fine tune the way to import annotations. More \
+on that in the :doc:`parameter description section</parameter_description>` of this guide.
+
+
+The Key-Value pairs were added to 5 datasets out of the 11 present in this project, as expected. \
+We also obtain three sets of Key-Value pairs, one for each namespace used in the csv.
+
+.. figure:: images/annotation_07_KV_import_printout.png
+
+  *The script output (5 entries in the csv matched to 5 dataset out of 11) and the\
+  resulting key-value pairs annotation.*
+..
+
+Converting the Key-Value pairs namespace
+----------------------------------------
+
+Key-Value pairs are grouped under categories called namespaces. This allows \
+flexibility to distinguish or isolate annotations, such as exporting or deleting \
+only those associated with a specific namespace.
+
+.. note::
+   In fact, if you created Key-Value pairs in OMERO.web, you have used \
+   namespaces without noticing it: OMERO assigns by default the \
+   "client namespace" (``openmicroscopy.org/omero/client/mapAnnotation`` in full)\
+   , a special namespace recognized by OMERO.web allowing to edit them.
```

Comment:
```suggestion
   , a special namespace recognized by OMERO.web.
```

You are explaining about the editing capability again just in the sentence below the note.